src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApi.java

```diff
@@ -355,7 +355,13 @@ public CaseDetails submitEventForAttachScannedDocs(
             );
         } catch (FeignException e) {
             throw new CcdCallException(
-                String.format("Could not attach documents for case ref: %s Error: %s", caseRef, e.status()), e
+                String.format(
+                    "Could not attach documents for case ref: %s, CCD response: %s, Error: %s",
+                    caseRef,
+                    e.contentUTF8(),
+                    e.status()
+                ),
+                e
```

Comment:
I know it's merged already, but best to put the response content last (it may be long) and label status with `Response status` instead of `Error`. I can do it if you agree @apereverzin

Reply:
@lgonczar Do not worry, I will raise another PR

Reply:
> I know it's merged already

was a bit ninja merge :(. i saw content being in the middle but didn't consider status to be super important as such - hence let it fly